omnibus/Gemfile.lock

```diff
@@ -1,9 +1,9 @@
 GIT
   remote: https://github.com/chef/omnibus-software.git
-  revision: 225e3576c48fcd0155f6049cb032b2370eccf29a
+  revision: 6d78999c60596386e25d02d630f379aac8ff1831
```

Comment:
Looks like the sha on omnibus-software is: 6a1c8896edf6c5606facc83b75ecaacf22019e2b https://github.com/chef/omnibus-software/commits/main

Reply:
Thanks.